### ISSUE TITLE
Allow a configurable cache-control header for redirects

### ISSUE THREAD

**Issue by mingard** [dadi/web#95]
Currently it is not possible to add further headers to a redirect. 
The `Cache-Control` header would help ensure a redirect wasn't cached by the browser.

```js
res.writeHead(301, {
  'Location': '/foo/bar',
  'Cache-Control': 'no-cache'
})
```

**Comment by jimlambie** [comment]
This is available now in v1.6.15

It uses the existing cacheControl config property, just add a value for each redirect type, for example:

```
  "headers": {
    "cacheControl": {
      "text/css": "public, max-age=3600",
      "301": "no-cache"
    }
  }
```

**Comment by hydroes** [comment]
// adding this here so we know how to implement.
res.writeHead(301, {"Location": "some.location"});
return res.end();
